source/CCustomOpcodeSystem.cpp

```diff
@@ -1706,22 +1701,28 @@ namespace CLEO {
 		// parse module reference text
 		if (moduleTxt != nullptr)
 		{
-			std::string str(moduleTxt);
+			std::string_view str(moduleTxt);
 			auto pos = str.find('@');
 			if (pos == str.npos)
 			{
 				std::string err(128, '\0');
-				sprintf(err.data(), "Invalid module reference '%s' in 0AB1 opcode in script '%s'", str.c_str(), thread->GetScriptFileName());
+				sprintf(err.data(), "Invalid module reference '%s' in 0AB1 opcode in script '%s'", moduleTxt, thread->GetScriptFileName());
 				Error(err.data());
 				return OR_INTERRUPT;
 			}
-			str[pos] = '\0'; // split into two texts
+			std::string_view strExport = str.substr(0, pos);
+			std::string_view strModule = str.substr(pos + 1);
 
 			// get module's absolute path
-			std::string modulePath(&str[pos + 1]);
-			modulePath = ResolvePath(modulePath.c_str(), thread->GetScriptFileDir());
+			std::string modulePath;
+			if (strModule.length() < 2 || strModule[1] != ':') // not starting with drive letter
+			{
+				modulePath = "2:\\"; // current script's location

```

Comment:
please make constants for new virtual drives (0:\\, 1:\\, etc)

Reply:
Done